src/lib/util.ts

```
@@ -0,0 +1,20 @@
+// Deprecated method usage
+import { deprecate } from "util";
+
+export const formatDate = (date: Date) => {
+  return date.toLocaleDateString();
+};
+
+// Using 'any' type
+export const parseJSON = (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)